supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala

```diff
@@ -451,7 +445,7 @@ class JesAsyncBackendJobExecutionActor(override val jobDescriptor: BackendJobDes
             FailedNonRetryableExecutionHandle(e).future
           case Failure(e: Exception) =>
             // Log exceptions and return the original handle to try again.
-            log.warning("Caught exception, retrying: " + e.getMessage, e)
+            log.warning(s"Caught exception, retrying: $e: " + e.getStackTrace.mkString(System.lineSeparator + " at: "), e)
```

Comment:
_Sorry, I don't know why akka's logging is different yet. If anyone has a good blog post I'd love to see it... anyways..._

I suspect you want:
```scala
log.warning("Caught exception, retrying: {}: {}", e, e.getStackTrace.mkString(System.lineSeparator + " at: "))
```

Separate from slf4j, Akka logging has its own [syntax](http://doc.akka.io/docs/akka/current/scala/logging.html#How_to_Log) for formatting messages. In the original line, it's passing an extra argument into "[Message template with 1 replacement argument.](http://doc.akka.io/api/akka/2.3.12/index.html#akka.event.LoggingAdapter@warning(template:String,arg1:Any):Unit)" Based on the syntax documentation this should end up with a possibly cryptic log message, as:

> Giving more arguments as there are placeholders results in a warning being appended to the log statement (i.e. on the same line with the same severity).

The above goes for any other akka log calls as well.